testsuite/cluster-test/src/suite.rs

```diff
@@ -79,11 +79,27 @@ impl ExperimentSuite {
         Self { experiments }
     }
 
+    fn new_land_blocking_compat_suite(cluster: &Cluster) -> Self {
+        let count: usize = env::var("BATCH_SIZE").unwrap().parse().unwrap();
```

Comment:
1) You probably can use some default value if BATCH_SIZE is not set instead of failing - I would suggest `cluster.validator_instances().len() / 2`.

2) `.unwrap` is usually undesired, because it is hard to debug(with few exceptions). Best practice here is probably changing this fn to return `Result<Self>` and returning error(via `bail!("Clear error message")`, for example) if parsing fails.